index.js

```diff
@@ -314,6 +314,123 @@ var handleCloudWatch = function(event, context) {
   return _.merge(slackMessage, baseSlackMessage);
 };
 
+var handleConfigCompliance = function(event, context) {
+  var timestamp = (new Date(event.Records[0].Sns.Timestamp)).getTime()/1000;
+  var message = JSON.parse(event.Records[0].Sns.Message);
+  var region = event.Records[0].EventSubscriptionArn.split(":")[3];
+  var ruleRegion = message.region;
+  var accountId = message.account;
+  var accountIdMap = JSON.parse(config.awsAccountMap).accounts
+  var accountName = message.AWSAccountId;
+  var subject = "AWS Config Rule Compliance State Change Notification";
+  var configRuleName = message.detail.configRuleName;
+  var resourceType = message.detail.resourceType;
+  var resourceId = message.detail.resourceId;
+  var newComplianceType = message.detail.newEvaluationResult.complianceType;
+  var color = "warning";
+
+  if (newComplianceType === "NON_COMPLIANT") {
+      color = "danger";
+  } else if (newComplianceType === "COMPLIANT") {
+      color = "good";
+  }
+  
+  for (var i=0; i< accountIdMap.length; i++) {
+    if (accountIdMap[i].accountId === accountId) {
+      accountName = accountIdMap[i].name;
+      break;
+    }
+  }
```

Comment:
could be ```_.find(accountIdMap, { accountId: accountId }).name```, but readability is debatable. Also this code is repeated twice so consider extracting to a function if used again